src/main/resources/messages/LanguageServerBundle.properties

```diff
@@ -37,6 +37,8 @@ language.server.tab.configuration.server=Server
 language.server.tab.configuration.client=Client
 language.server.configuration=Configuration:
 language.server.initializationOptions=Initialization Options:
+language.server.configuration.json.schema.associate=Associate with Json Schema

```

Comment:
`Json` should be `JSON` in all of these labels since that's the actual language name (**J**ava**S**cript **O**bject **N**otation).

Reply:
fixed